pkg/api/interceptor.go

```diff
@@ -96,17 +96,17 @@ func (wa *WalletAuthorizer) requiresAuthorization(req interface{}) bool {
 func (wa *WalletAuthorizer) getWallet(ctx context.Context) (types.WalletAddr, error) {
 	md, ok := metadata.FromIncomingContext(ctx)
 	if !ok {
-		return "", status.Errorf(codes.Unauthenticated, "metadata is not provided")
+		return "", status.Error(codes.Unauthenticated, "metadata is not provided")
```

Comment:
Something has been updated in our linter and it seems to have become stricter on using `Errorf` for errors that don't require formatting